src/pages/Liquidated.tsx

```diff
@@ -64,11 +64,14 @@ export function Liquidated() {
 
   const graphDataList = populateMissingDays(graphDataMap, GRAPH_DAYS);
 
-  const summedGraphDataList = graphDataList.map((item) => ({
-    ...item,
-    active: sum(Object.values(item.active)),
-    liquidated: sum(Object.values(item.active)),
-  }));
+  const summedGraphDataList = graphDataList.map((item) => {
+    const total = item.active ? sum(Object.values(item.active)) : 0;
```

Comment:
change default to Nan or undefined. we dont want to falsely indicate 0 as a possible value

Reply:
It's graph data. 0 is a valid fallback. The graph shows nothing. With NaN it's the opposite.